Back/src/app.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Controller, Get } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+const getComidas = async() => {
+  const comidas = await prisma.comida.findMany({
+    include: {
+      ingredientes: {
+        select : {
+          nombre: true
+        }
+      }
+    },
+  });
+
+  const comidasSort = comidas.map(comida => 
+    ({...comida, ingredientes: comida.ingredientes.map(ingrediente => 
+      ingrediente.nombre),}));
+
+  return comidasSort;
+
+}
```

Comment:
This should go into a service file, e.g. app.service.ts. Thats one of the aspects of hexagonal architecture